### ISSUE TITLE
Add a play button

### ISSUE THREAD

**Issue by placoderm** [gpodder/gpodder#1277]
I've been using this app for a couple of years and really love it. I appreciate all the hard work people have put into it.

Why isn't there a play button? To me it seems so completely obvious that an app for downloading audio files should have a button to click to play them. 

There is this issue https://github.com/gpodder/gpodder/issues/228 about the default double click behaviour, but to me it misses the point that there should just be a play button. 

Honestly I think there should be a download button for each episode as well, but I'd settle for a play button.

Thanks again to everyone who has worked to make this app as great as it is.

**Comment by auouymous** [comment]
Other than the play and download buttons in View..Toolbar?

**Comment by placoderm** [comment]
Yes. Otherwise you have to do two clicks to play. Select it then play it. 

I totally get it that this is a minor problem.

**Comment by auouymous** [comment]
After the release, I would like to allow customization of keys and double clicking. The defaults won't change because people have been using them for years, but double click could be changed to play or stream.

There could also be an option (off by default) to play when the play icon is clicked, or stream if the delete icon or no icon. Clicking anywhere in the left side that activates the tooltip would invoke play or stream.

Another option (off by default) could add a download icon column next to the current icon, if possible. When not downloaded, the icon would appear and could be clicked to download.
